test/instrumentation/transaction.test.js

```diff
@@ -315,7 +315,7 @@ test('#_encode() - ended', function (t) {
   t.ok(payload.duration > 0)
   t.strictEqual(payload.timestamp, trans._timer.start)
   t.strictEqual(payload.result, 'success')
-  t.deepEqual(payload.context, { user: {}, tags: {}, custom: {} })
+  t.deepEqual(payload.context, { user: {}, tags: {}, custom: {}, service: {}, cloud: {}, message: {} })
```

Comment:
Will all our transactions now always have these extra fields? I think it would be nice to minimize that. It could be a separate PR, if you prefer.

Oh actually, if you take the suggestion in transaction.js to *not* have the `|| {}`, then I think this will clear up.

Reply:
Added a separate issue to address this: https://github.com/elastic/apm-agent-nodejs/issues/2416